decidim-core/app/commands/decidim/create_omniauth_registration.rb

```diff
@@ -91,6 +92,20 @@ def signature_valid?
       signature = OmniauthRegistrationForm.create_signature(form.provider, form.uid)
       form.oauth_signature == signature
     end
+
+    def trigger_omniauth_registration(identity)
+      ActiveSupport::Notifications.publish(
+        "decidim.events.user.omniauth_registration",
+        user_id: @user.id,
+        identity_id: identity.id,
+        provider: form.provider,
+        uid: form.uid,
+        email: form.email,
+        name: form.name,
+        nickname: form.normalized_nickname,
+        avatar_url: form.avatar_url
```

Comment:
Maybe it would be good to also include the raw hash given by the OAuth server? Maybe there's some data there that we don't use for the user or the identity but it's useful for other services.